libdrgn/python/program.c

```diff
@@ -839,6 +839,17 @@ static PyObject *Program_get_platform(Program *self, void *arg)
 		Py_RETURN_NONE;
 }
 
+static PyObject *Program_get_language(Program *self)
+{
+	const struct drgn_language *lang;
+
+	lang = drgn_program_lang(&self->prog);
+	if (lang)
+		return Language_wrap(lang);
+	else
+		Py_RETURN_NONE;
```

Comment:
This won't ever be true since we fallback to the default language, correct? Which is the behavior I think we want.

Reply:
I think this shouldn't ever happen, but I wanted to have this fallback just in case there was some bug. But I suppose failing hard is better so we catch issues early. I'll remove this.